docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/vmware-tanzu.md

```diff
@@ -166,19 +180,19 @@ If you choose to use VMware Tanzu Application Service, before creating the servi
 - The YugabyteDB tile is installed in your PCF marketplace.
 - The cloud provider is configured in the YugabyteDB Anywhere instance in your PCF environment .
 
-### Creating a YugabyteDB Service Instance
+### Create a YugabyteDB Service instance
 
 You can create a YugabyteDB service instance via the App Manager UI or Cloud Foundry (cf) command-line interface (CLI).
```

Comment:
Cloud Foundry docs do it this way: Cloud Foundry Command Line Interface (cf CLI); cf CLI is used further down
(ref https://docs.cloudfoundry.org/cf-cli/)